packaging/debian/README.md

```diff
@@ -3,26 +3,39 @@ Ansible Debian Package
 
 To create an Ansible DEB package:
 
-    sudo apt-get install python-paramiko python-yaml python-jinja2 python-httplib2 python-setuptools python-six sshpass
-    sudo apt-get install cdbs debhelper git-core reprepro fakeroot asciidoc devscripts docbook-xml xsltproc libxml2-utils
-    sudo apt-get install dh-python build-essential
-    git clone git://github.com/ansible/ansible.git
-    cd ansible
-    make deb
+__Note__: You must either run as root or have write access to `/var/cache/pbuilder`
```

Comment:
@shanemcd Write access to `/var/cache/pbuilder` isn't enough. The build process fails with this error when not running as root:

```
E: debootstrap can only run as root
```